### PR TITLE
Restore ResNet checkpoints

### PR DESCRIPTION
**Pull request by robieta**
We haven't had pretrained checkpoints since the recent topology changes. This adds them back.

### CODE REVIEW

File: official/resnet/README.md [hunk 1]
```diff
@@ -60,3 +60,18 @@ num_gpus:
...
+
+### Pre-trained model
+You can download 190 MB pre-trained versions of ResNet-50. Reported accuracies are top-1 single-crop accuracy for the ImageNet validation set. Simply download and uncompress the file, and point the model to the extracted directory using the `--model_dir` flag.
+
+ResNet-50 v2 (Accuracy 76.05%):
+* [Checkpoint](http://download.tensorflow.org/models/official/20180601_resnet_v2_imagenet_checkpoint.tar.gz)
+* [SavedModel](http://download.tensorflow.org/models/official/20180601_resnet_v2_imagenet_savedmodel.tar.gz)
+
+ResNet-50 v2 (fp16, Accuracy 75.56%):
+* [Checkpoint](http://download.tensorflow.org/models/official/20180601_resnet_v2_fp16_imagenet_checkpoint.tar.gz)
```
Comment (qlzh727) [inline official/resnet/README.md]: Is there any reason to only provide fp16 in v2?

Comment (robieta) [reply official/resnet/README.md]: It's what I had handy from other fp16 testing.

Comment (qlzh727) [reply official/resnet/README.md]: It will be nice to make them consistent, have fp16 for both or neither.

Comment (robieta) [reply official/resnet/README.md]: Alright, I'll fire off a v1 fp16 run at some point.

Comment (robieta) [reply official/resnet/README.md]: Looks like v1 fp16 still has numerical issues, so no checkpoint for that configuration.

**Pull request merged**